pkg/parser/extended_syntax_parser.y

```diff
@@ -450,6 +450,12 @@ func parseSQLFlowStmt(s string) (r *SQLFlowSelectStmt, e error) {
 	mu.Lock()
 	defer mu.Unlock()
 
-	extendedSyntaxParse(newLexer(s))  // extendedSyntaxParse is auto generated.
-	return parseResult, nil
+	parseResult = nil // Important! Clear out result from previous call.
+        lex := newLexer(s)
```

Comment:
Indent error?

Reply:
Yes. Go fmt doesn't handle .y files. I will fire another PR with tools that can auto-indent .y files.